packages/app-accounts/src/Overview.tsx

```diff
@@ -26,10 +26,16 @@ type State = {
 };
 
 class Overview extends React.PureComponent<Props, State> {
-  state: State = {
-    isCreateOpen: false,
-    isImportOpen: false
-  };
+  constructor (props: Props) {
+    super(props);
+    const { state = {} } = this.props.location;
+    const { isCreateOpen } = state;
+
+    this.state = {
+      isCreateOpen: isCreateOpen,
```

Comment:
isCreateOpen may be undefined, so should rather be `isCreateOpen: !!isCreateOpen`